cnd/src/main.rs

```diff
@@ -45,10 +49,18 @@ fn main() -> anyhow::Result<()> {
 
     let bitcoin_connector = {
         let config::Bitcoin { node_url, network } = settings.clone().bitcoin;
-        BitcoindConnector::new(node_url, network)?
+
+        BitcoinCache {
+            inner: BitcoindConnector::new(node_url, network)?,
+            cache: Arc::new(Mutex::new(lru::LruCache::new(100))),
```

Comment:
You could provide a constructor `BitcoinCache::new(inner, capacity)`. That would make this more ergonomic because it is always the same anyway :)

Reply:
Can we have a const as well please instead of a magic number.  Eventually we will likely want to expose this value as a config option.